docs/how-to/configure-jenkins-memory-usage.md

```diff
@@ -0,0 +1,12 @@
+# How to control heap memory of the jenkins-k8s-operator charm
+The [jenkins-k8s-operator](https://github.com/canonical/jenkins-k8s-operator) charm uses [juju constraints](https://juju.is/docs/juju/constraint) to limit the amount of memory a charm can use. To deploy the charm with constraints, use the `--constraints "<key>=<value>"` option when running `juju deploy`:
+```bash
+juju deploy jenkins-k8s --channel=latest/edge --constraints "mem=2048M"
+```
+To change this value after deployment, use the `set-constraints` command
+```bash
+juju set-constraints jenkins-k8s "mem=4096M"
```

Comment:
I didnt find it in the docs but note that this will affect every container in the pod. So both charm and Jenkins containers will have the same request/limits.

Reference:
https://github.com/juju/juju/pull/14464

Reply:
Good point, I'll reflect that in the docs